bundles/mapping/mapwfs2/plugin/WfsVectorLayerPlugin/impl/MvtLayerHandler.ol.js

```diff
@@ -16,6 +18,16 @@ export class MvtLayerHandler extends AbstractLayerHandler {
         super(layerPlugin);
         this._log = Oskari.log('WfsMvtLayerPlugin');
         this.localization = Oskari.getMsg.bind(null, 'MapWfs2');
+        this.throttledUpdates = new Map();
+        this.tileGrid = this._getTileGrid();
+    }
+    _getTileGrid () {
+        const tileGrid = {
+            extent: [-548576, 6291456, 1548576, 8388608],
+            resolutions: [8192, 4096, 2048, 1024, 512, 256, 128, 64, 32, 16, 8, 4, 2, 1, 0.5, 0.25],
+            tileSize: [256, 256]
+        };
+        return new olTileGrid(tileGrid);
```

Comment:
This change will result in this only working on EPSG:3067

Reply:
Refactored to use layer's tile grid configuration when exists.